structurizr/workspace.dsl

```diff
@@ -0,0 +1,260 @@
+            containerFoWeb -> containerFoAzureCosmos "stores user session data"
+			containerFoWeb -> containerKeyVault "retrieves secrets"
+            
+			// Appeals API
+            containerFoApi -> containerFoAzureSql "stores data"
+            containerFoApi -> containerFoAzureCosmos "stores data"
+            containerFoApi -> containerFoAuthServer "validates auth tokens"
+            containerFoApi -> componentCaseCmdTopic "informs back office of new appeals"
+            containerFoApi -> componentLpaqCmdTopic "informs back office of new lpaqs"
+			containerFoApi -> containerKeyVault "retrieves secrets"
+
+			// Docs API
+            containerFoDocsApi -> containerFoAzureSql "stores data"
+            containerFoDocsApi -> containerFoAzureCosmos "stores data"
+            containerFoDocsApi -> containerFoAuthServer "validates auth tokens"
+            containerFoDocsApi -> containerFoFileStorage "stores/retrieves documents"
+            containerFoDocsApi -> containerBoFileStorage "retrives BO docs"
+			containerFoDocsApi -> containerKeyVault "retrieves secrets"
+
+
+			// Notify
+            containerFoApi -> containerGovNotify "Sends emails"
+            containerFoAuthServer -> containerGovNotify "Sends emails"
+			containerBoWeb -> containerGovNotify "Sends emails"
+			containerGovNotify -> userLocalPlanningAuthority "Notifies the LPA"
+			containerGovNotify -> userAppellant "Notifies the appellant"
+			containerGovNotify -> userInterested "Notifies the interested party"
+			containerGovNotify -> userRule6 "Notifies the rule 6 interested party"
+
+			// clam av
+			containerFoClamAV -> systemClamAv "Gets latest definitions"
+
+			// service bus
+            containerFoApi -> componentCaseCmdTopic "Records appellant submissions"
+			containerFoApi -> componentLpaqCmdTopic "Records LPA submissions"
+
+			containerBoWeb -> componentCaseMessageTopic "Records changes to appeals"
+			containerBoWeb -> componentDocumentMessageTopic "Records changes to documents"
+			containerBoWeb -> componentServiceUserMessageTopic "Records changes to user information"
+			containerBoWeb -> componentEventMessageTopic "Records changes to site visits and other events"
+
+			// Integration functions
+			containerFunctionApp -> componentCaseMessageTopic "Poll for new messages"
+			containerFunctionApp -> componentDocumentMessageTopic "Poll for new messages"
+			containerFunctionApp -> componentServiceUserMessageTopic "Poll for new messages"
+			containerFunctionApp -> componentEventMessageTopic "Poll for new messages"
+			containerFunctionApp -> containerFoApi "forward service bus messages to API"
+
+            // legacy
+            userCaseOfficer -> systemLegacyBo "Manages cases"
+			userInspector -> systemLegacyBo "Decides cases"
+            containerFoApi -> systemLegacyIntegration "Scheduled job sends appeals to Horizon Wrapper"
+            systemLegacyIntegration -> systemLegacyBo "Forwards appeals to legacy back office"
+		}
+	}
+
+	views {
```

Comment:
Just one comment from me - to generate static diagrams with `ghcr.io/avisi-cloud/structurizr-site-generatr` some views are needed on systems, containers and components... e.g.: https://github.com/Planning-Inspectorate/appeals-back-office/blob/8d1aef7ef45b8b20d22a78890ca55faec56b1560/workspace.dsl#L141